pandas/tests/test_indexing.py

```diff
@@ -788,6 +789,50 @@ def test_loc_setitem_multiindex(self):
         result = df.loc[(t,n),'X']
         self.assertEqual(result,3)
 
+    def test_ix_setitem_multiindex(self):
+        # GH5206
+        df = pd.DataFrame(np.random.random((5, 5)), columns='A,B,C,D,E'.split(','))
```

Comment:
I'm also going to change this test to not use `np.random` I think, as I note my row selection is `<0.5` below, and thus the test inputs will be different between runs, which seems like a bad smell.